stdlib/tests/math/u64_mod.rs

```diff
@@ -517,7 +518,10 @@ fn checked_and_fail() {
         end";
 
     let test = build_test!(source, &[a0, a1, b0, b1]);
-    test.expect_error(TestError::ExecutionError("NotU32Value"));
+    test.expect_error(TestError::ExecutionError(ExecutionError::NotU32Value(
+        Felt::new(b0),
+        Felt::new(0),
```

Comment:
minor nit: here and in other similar places, we should be able use `Felt::ZERO` (or even just `ZERO`) instead of `Felt::new(0)`.

Reply:
Updated all of the tests so that they use ZERO (from test-utils) instead of Felt::new(0).